Test/civl/inductive-sequentialization/paxos/PaxosSeq.bpl

```diff
@@ -29,12 +29,12 @@ modifies pendingAsyncs;
   pendingAsyncs := PAs;
 }
 
-function {:inline} FirstCasePAs(k: int, numRounds: int) : [PA]int
+function {:inline} StartRoundPAs(k: int, numRounds: int) : [PA]int
 {
   (lambda pa: PA :: if (is#A_StartRound(pa) && round#A_StartRound(pa) == round_lin#A_StartRound(pa) && k < round#A_StartRound(pa) && round#A_StartRound(pa) <= numRounds) then 1 else 0)
 }
 
-function {:inline} SecondCasePAs(k: int, m: Node, numRounds: int) : [PA]int
+function {:inline} StartRoundPlusJoinPlusProposePAs(k: int, m: Node, numRounds: int) : [PA]int
```

Comment:
I don't love these names, but they are better than the generic ones :)

I think it would really help to add comments that explain the sequentialization and what intermediate states the helper functions express. I can do that in a separate PR.

Reply:
Ok, please go ahead and add comments in your PR after this one lands.